<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Organisation</class>
 <widget class="QMainWindow" name="Organisation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>346</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>12</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Оплата услуг</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <activeon>:/logo.ico</activeon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>12</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Телефония</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="phoneBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="phoneButton">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>12</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Выбрать</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Коммунальные платежи</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="komBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="komButton">
          <property name="text">
           <string>Выбрать</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Интернет, Телевидение</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="internetBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="internetButton">
          <property name="text">
           <string>Выбрать</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Популярные платежи</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="famousBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="famousButton">
          <property name="text">
           <string>Выбрать</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="backButton">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
